primitives/src/lib.rs

```diff
@@ -65,3 +69,33 @@ pub type Index = u32;
 pub type Block = sp_runtime::generic::Block<Header, sp_runtime::OpaqueExtrinsic>;
 /// Index of a transaction in the chain.
 pub type Nonce = u32;
+/// Revert opt code. It's inserted at the precompile addresses, to make them functional in EVM.
```

Comment:
I could create a separate evm module for these primitives.

Reply:
Yes please create a separate module.